src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -858,6 +858,22 @@ namespace winrt::TerminalApp::implementation
             }
         });
 
+        newTabImpl->TabRenamerDeactivated([weakThis{ get_weak() }](auto&& /*s*/, auto&& /*e*/) {
+            if (const auto page{ weakThis.get() })
+            {
+                if (!page->_newTabButton.Flyout().IsOpen())
+                {
+                    if (const auto tab{ page->_GetFocusedTabImpl() })
+                    {
+                        if (const auto control{ tab->GetActiveTerminalControl() })
+                        {
+                            control.Focus(FocusState::Programmatic);
+                        }
```

Comment:
Nit: should we just do `tab->Focus(FocusState::Programmatic)` to keep it consistent with the rest of the file?

Reply:
Love it! Thanks!